DigitalLearningSolutions.Web/Controllers/ResetPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Services;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class ResetPasswordController : Controller
+    {
+        private readonly IPasswordResetService passwordResetService;
+        public ResetPasswordController(IPasswordResetService passwordResetService)
+        {
+            this.passwordResetService = passwordResetService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Index(string email, string code)
+        {
+            if (string.IsNullOrEmpty(email) || string.IsNullOrEmpty(code))

```

Comment:
I'm wondering whether we should also redirect the user if they are logged in  🤔

Reply:
Hmm, yeah, I'm not sure. I think we should be consistent with the ForgotPassword page, which does redirect to Login.